README.md

```diff
@@ -112,13 +109,8 @@ A great place to get involved or ask questions is [#sig-cluster-lifecycle](https
  - Look at our [other interesting modes](/docs/commands.md#other-interesting-modes)
  - Full command line interface [documentation](/docs/cli/kops.md)
 
-## History
-
```

Comment:
I am fine with cleaning this up for now.. we can (and should) address the changelog during our next drop (1.5.0)